### PR TITLE
Re-add Storybook entries for ProjectsList

### PR DESCRIPTION
**Pull request by bencodeorg**
Re-enable a Storybook entry for a component that shows a table of student projects on the teacher dashboard. Remove a entries for a parent component (`SectionProjectsList`), which a) has unit tests and b) basically just renders the `ProjectsList` component and a dropdown above it to select a student.

## Testing story

New Storybook entries:

image

**With Thumbnails**

image

**Without Thumbnails**

image

### CODE REVIEW

File: apps/src/templates/projects/ProjectsList.story.jsx [hunk 1]
```diff
@@ -27,41 +32,26 @@ const STUB_PROJECTS_DATA = [
...
-        <ProjectsList
-          locale={'es-MX'}
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={true}
-        />
-      )
-    }
-  ]);
+const DEFAULT_ARGS = {
```
Comment (epeach) [inline apps/src/templates/projects/ProjectsList.story.jsx]: I'm curious if there is a benefit to having these DEFAULT_ARGS or if we can just put them in the template itself, like
```
<ProjectsList
  projectsData: STUB_PROJECTS_DATA,
  studioUrlPrefix: 'https://studio.code.org',
  showProjectThumbnails: true
  {...args}
/>
```

File: apps/src/templates/projects/ProjectsList.story.jsx [hunk 2]
```diff
@@ -27,41 +32,26 @@ const STUB_PROJECTS_DATA = [
...
+  projectsData: STUB_PROJECTS_DATA,
+  studioUrlPrefix: 'https://studio.code.org',
+  showProjectThumbnails: true
+};
+
+const Template = args => <ProjectsList {...args} />;
+
+// Name of export determines what is shown in storybook entry
```
Comment (epeach) [inline apps/src/templates/projects/ProjectsList.story.jsx]: Intentionally left in?

**Review by epeach — COMMENTED** [review]
I agree with your call to remove the story for SectionProjectsList. I left a feed comments on the ProjectsList translation. Let me know if you have any questions!

File: apps/src/templates/projects/ProjectsList.story.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/templates/projects/ProjectsList.story.jsx]: Nope, but with the refactor I now need it. So, now, yes!

File: apps/src/templates/projects/ProjectsList.story.jsx (hunk 1 above)
Comment (bencodeorg) [reply apps/src/templates/projects/ProjectsList.story.jsx]: What I had was closer to the [Storybook docs](https://storybook.js.org/docs/react/writing-stories/introduction), but I like yours better. Updated!

File: apps/src/templates/projects/ProjectsList.story.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/templates/projects/ProjectsList.story.jsx]: oh nvm, just saw in [this PR](https://github.com/code-dot-org/code-dot-org/pull/49124) a similar thing without the linting exception. Turns out my editor was complaining but it committed without issue. Removed!

**Review by epeach — APPROVED** [review]
LGTM!

**Pull request merged**